sdk/python/kfp_tekton/compiler/_op_to_template.py

```diff
@@ -319,7 +319,32 @@ def _process_output_artifacts(outputs_dict: Dict[Text, Any],
         mounted_artifact_paths = []
         for artifact in outputs_dict['artifacts']:
             artifact_name = artifact_to_result_mapping.get(artifact['name'], artifact['name'])
-            if artifact['name'] in replaced_param_list:
+            if artifact['path'] == 'logs_to_S3':
+                copy_artifacts_step['script'] = copy_artifacts_step['script'] + \
+                    'mc cp /var/log/containers/$TASKRUN*$NAMESPACE*.log' + \
```

Comment:
Also somehow the ibm cloud cluster doesn't allow copying file directly.
```
mc: <ERROR> Failed to copy `/var/log/containers/parallel-pipeline-c3fae-gcs-download-j7t2j-pod-ll889_kubeflow_step-main-f3a0a7eaa1bb7a4678eb8877777a59cde91d0c8571c81dded25ff60b1e29e7c1.log`. You did not provide the number of bytes specified by the Content-Length HTTP header.
mc: <ERROR> Failed to copy `/var/log/containers/parallel-pipeline-c3fae-gcs-download-j7t2j-pod-ll889_kubeflow_step-copy-artifacts-7e5fe4413051d1984fc1135d1b0fb7300258b6de625fa9aa3b998365213e6354.log`. You did not provide the number of bytes specified by the Content-Length HTTP header.
Total: 0 B, Transferred: 1.05 KiB, Speed: 2 B/s
```

So I have to convert this part into something like
```shell
cat /var/log/containers/$TASKRUN*$NAMESPACE*.log > /tmp/main-log
tar -cvzf main-log.tgz /tmp/main-log
mc cp main-log.tgz storage/mlpipeline/artifacts/$PIPELINERUN/$PIPELINETASK/
```

Reply:
I am wonder this is an minio issue, similar issues logged before:
https://github.com/minio/minio/issues/6139
https://github.com/minio/minio/pull/6489

**Anyway, address it in the code here.** 
But unfortunately, my lab cluster has some issues these days, I cannot UT for it right now, will try it later.